File: src/nodemodule/basic/TestCalc.js
var add = require('./calc.js').add
console.log(add(20,40))

var {sub,Calc,calc,value} = require('./calc')

console.log(sub(30,40))
c = new Calc()
console.log(c.mul(30,40))
console.log(calc.mul(12,43))
console.log(value)

File: src/staticlib_nativemodule/cpp/TestCalc.cpp
#include "Calc.h"
#include <iostream>

using namespace std;

int main(int argc, char *argv[]) {
    double a = 22;
    double b = 6;

    cout << "a + b = " << Calc::add(a,b) << endl;
    cout << "a - b = " << Calc::sub(a,b) << endl;
    cout << "a * b = " << Calc::mul(a,b) << endl;
    cout << "a / b = " << Calc::div(a,b) << endl;
    return 0;
}